### PR TITLE
Kill Addrindexrs

### PR DESCRIPTION
**Pull request by ouziel-slama**
- Mock get_oldest_tx()
- use Electr to get utxos and address history

------

* [x] Double-check the spelling and grammar of all strings, code comments, etc.
* [x] Double-check that all code is deterministic that needs to be
* [x] Add tests to cover any new or revised logic
* [x] Ensure that the test suite passes
* [x] Update the project [release notes](release-notes/)
* [x] Update the project documentation, as appropriate, with a corresponding Pull Request in the [Documentation repository](https://github.com/CounterpartyXCP/Documentation)

### CODE REVIEW

File: release-notes/release-notes-v10.9.0.md [hunk 1]
```diff
@@ -18,6 +18,7 @@
 - Add a timeout to parse mempool transaction from ZMQ
 - Add cache for unsupported transactions when parsing raw 
 - Refactor bootstrap: clean code, use `zstd` instead `gzip`, decompress and verify signature in parallel
+- Remove Addrindexrs dependency: mock `get_oldest_tx()`, use Electr to get utxos and address history
```
Comment (adamkrellenstein) [inline release-notes/release-notes-v10.9.0.md:21]: -> "to get utxos and address history for tx composition"

File: counterparty-core/counterpartycore/server.py [hunk 2]
```diff
@@ -618,6 +585,16 @@ def initialise_config(
     config.GUNICORN_THREADS_PER_WORKER = gunicorn_threads_per_worker
     config.GUNICORN_WORKERS = gunicorn_workers
 
+    if electr_url:
```
Comment (adamkrellenstein) [inline counterparty-core/counterpartycore/server.py:588]: `electr` -> `electrs`

File: release-notes/release-notes-v10.9.0.md (hunk 1 above)
Comment (ouziel-slama) [reply release-notes/release-notes-v10.9.0.md:21]: fixed

File: counterparty-core/counterpartycore/server.py (hunk 2 above)
Comment (ouziel-slama) [reply counterparty-core/counterpartycore/server.py]: fixed

**Pull request merged**